docs/addons/karpenter.md

```diff
@@ -49,7 +49,9 @@ const karpenterAddonProps = {
   }],
   amiFamily: "AL2",
   consolidation: { enabled: true },
-  weight: 20
+  ttlSecondsUntilExpired: 360,
```

Comment:
Any user setting `ttlSecondsUntilExpired` this low will experience severe churn. The example from the [Karpenter docs](https://karpenter.sh/v0.20.0/concepts/provisioning/) uses `2592000` (30 days).

I suggest using the same value for the documentation here.

Reply:
By default is not set, so never expires, however as a doc sample agree with @aceat64, should be set to a more reasonable number to avoid blind copy/paste resulting in churn.

Reply:
addressed